src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/4.diff.sql

```diff
@@ -0,0 +1,78 @@
+/*************************************************************
+    Search Parameter Status Information
+**************************************************************/
+
+ALTER TABLE dbo.SearchParam
+ADD
+    Status varchar(10) NOT NULL,
+    LastUpdated datetimeoffset(7) NOT NULL,
+    IsPartiallySupported bit NOT NULL
+
+CREATE TYPE dbo.SearchParamTableType_1 AS TABLE
+(
+    Uri varchar(128) COLLATE Latin1_General_100_CS_AS NOT NULL,
+    Status varchar(10) NOT NULL,
+    IsPartiallySupported bit NOT NULL
+)
+
+GO
+
+/*************************************************************
+    Stored procedures for search parameter information
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     Gets all the search parameters and their statuses.
+--
+-- DESCRIPTION
+--     Retrieves and returns select contents from the search parameter table.
+--
+-- RETURN VALUE
+--     The search parameters and their statuses.
+--
+CREATE PROCEDURE dbo.GetSearchParamStatuses
+AS
+    SET NOCOUNT ON
+
+    SELECT Uri, Status, LastUpdated, IsPartiallySupported FROM dbo.SearchParam
+GO
+
+--
+-- STORED PROCEDURE
+--     Given a set of search parameters, creates or updates the parameters.
+--
+-- DESCRIPTION
+--     If a parameter with a matching URI already exists in the table, it is updated.
+--     If not, a new entry is created.
+--
+-- PARAMETERS
+--     @searchParams
+--         * The updated or new search parameters
```

Comment:
Should it be The existing or new search parameters?

Reply:
Reworded this to be a bit more clear in 0bae8e3.